Pwm1.0

ryankapur opened the pull request:
Pls provide feedback on the Actuator class. 

Thanks.

ryankapur committed:
Modularized tracking code, fixed pwm import issue. Ready to test actuator.

ryankapur committed:
deleted:    ../README.md

ryankapur committed:
Syntactic changes to be committed:
	modified:   config.json
	modified:   ina219.py
	modified:   mppt.py

ryankapur committed:
deleted:    README.txt

ryankapur committed:
First draft of Actuator class:
	modified:   actuator_pwm.py

ryankapur committed:
renamed:    README.txt -> README.md

ryankapur committed:
Update README.md

ryankapur committed:
Modularized actuator code.
	modified:   actuator_pwm.py

ryankapur committed:
modified:   config.json

ryankapur committed:
Added first-draft functionality for second actuator.
	modified:   actuator_pwm.py
	modified:   config.json
	modified:   pwm_funcs.py
	deleted:    tracker.py
	modified:   tracker_funcs.py

ryankapur commented:
I finished the first draft of implementing the second actuator. I did not make a separate actuator class yet because I did not know what it entails yet seemed not applicable--yet.

I will review the law of cosines math I used on Monday with you--seems to check out; however, currently the current effectivePanningDistance is the same when azimuth is value x and negative value x...That is something we can discuss.

Regards,
Ryan

tripzero commented on python/actuator_pwm.py:
```diff
@@ -0,0 +1,22 @@
+maxActuatorHeight = 2 #TODO: Confirm maxActuatorHeight is equivalent for both actuators and both will use mraa pwm
+myPercent = effectiveActuatorHeight2 / maxActuatorHeight
+b = pwm.Actuator(3, maxActuatorHeight, myPercent, 700, True)
+b.set(myPercent, 700)
```
does this still work if set has only one argument: set(percent)?

Other than this, it looks good.

ryankapur committed:
Modularized code; working on datetime issue; func demoDay. Actuator vals are
malformed currently for altitude due to improper datetime. Must convert UTC<->
Pacific Time with an online module hopefully by tomorrow morning.

	modified:   actuator_pwm.py
	modified:   config.json
	modified:   pwm_funcs.py
	modified:   tracker_funcs.py
	modified:   tracking.py
